__tests__/app.test.js

```diff
@@ -203,4 +203,67 @@ describe("POST /api/articles/:article_id/comments", () => {
       expect(msg).toBe("Bad request")
     })
   })
+})
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("200: Updates the votes property with a new vote count and responds with the updated article", () => {
+    const newVoteCount = { inc_votes: 5 }
+    return request(app)
+    .patch("/api/articles/1")
+    .send(newVoteCount)
+    .expect(200)
+    .then(({ body: { article } }) => {
+      expect(article).toMatchObject({
+        article_id: 1,
+        title: "Living in the shadow of a great man",
+        topic: "mitch",
+        author: "butter_bridge",
+        body: "I find this existence challenging",
+        created_at: expect.any(String),
+        votes: 105,
+        article_img_url:
+          "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+      })
+    })
+  })
+  test("200: Updates the votes property correctly when passed a negative number", () => {
+    const newVoteCount = { inc_votes: -42 }
+    return request(app)
+    .patch("/api/articles/1")
+    .send(newVoteCount)
+    .expect(200)
+    .then(({ body: { article } }) => {
+      expect(article).toMatchObject({
+        article_id: 1,
+        title: "Living in the shadow of a great man",
+        topic: "mitch",
+        author: "butter_bridge",
+        body: "I find this existence challenging",
+        created_at: expect.any(String),
+        votes: 58,
+        article_img_url:
+          "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+      })
+    })
+  })
+  test("404: Responds with message when given a valid but non-existent article_id", () => {
+    const newVoteCount = { inc_votes: 5 }
+    return request(app)
+    .patch("/api/articles/999")
+    .send(newVoteCount)
+    .expect(404)
+    .then(({ body: { msg } }) => {
+      expect(msg).toBe("Article_id does not exist")
+    })
+  })
+  test("400: Responds with Bad Request when given an invalid article_id", () => {
+    const newVoteCount = { inc_votes: 5 }
+    return request(app)
+    .patch("/api/articles/not-an-article")
+    .send(newVoteCount)
+    .expect(400)
+    .then(({ body: { msg } }) => {
+      expect(msg).toBe("Bad request")
+    })
+  })
```

Comment:
These are great error tests, what if the value of `inc_votes` is not a number? What if that key doesn't exist? What if there are extra keys in there, how will you deal with that?